debezium-quarkus/extensions/outbox/deployment/src/main/java/io/debezium/quarkus/outbox/DebeziumOutboxConfig.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.quarkus.outbox;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+/**
+ *
+ * Configuration root class for Debezium Outbox pattern that defines the available user
+ * configuration options to customize this extension's behavior.
+ *
+ * @author Chris Cranford
+ */
+@ConfigRoot(phase = ConfigPhase.BUILD_TIME)
+public class DebeziumOutboxConfig {
+    /**
+     * The table name to be used for the outbox table
+     */
+    @ConfigItem(defaultValue = "OutboxEvent")
+    public String tableName;
+
+    /**
+     * The column name that contains the event id in the outbox table
+     */
+    @ConfigItem(defaultValue = "id")
+    public String columnIdName;
```

Comment:
`idName()`? Or `idColumnName()`?

Reply:
I'm inclined to think `idColumnName` and probably do that for all columns for consistency.